### PR TITLE
add removal logic

### PR DESCRIPTION
**Pull request by pschoffer**
Iterate over existing folders. If the folder should not be there (lower than the supported version or skipped version) remove it.

### CODE REVIEW

File: ci/update-jetpack.js [hunk 1]
```diff
@@ -25,6 +26,54 @@ function incrementVersion(version) {
...
+
+function compareInt(a,b) {
+    if ( a === b ) {
+        return 0;
+    }
+    if ( a > b ) {
+        return 1;
+    }
+    return -1;
+}
```
Comment (sjinks) [inline ci/update-jetpack.js]:
```js
return a - b;
```
?

File: ci/update-jetpack.js [hunk 2]
```diff
@@ -25,6 +26,54 @@ function incrementVersion(version) {
     return result;
 }
 
+function parseVersion( version ) {
+    return version.split(/[.-]/).map(part => isNaN(Number(part)) ? part : Number(part));
```
Comment (sjinks) [inline ci/update-jetpack.js:30]:
```suggestion
    return version.split(/[.-]/).map(part => isNaN(+part) ? part : +part);
```

**Review by sjinks — APPROVED** [review]

File: ci/update-jetpack.js (hunk 1 above)
Comment (pschoffer) [reply ci/update-jetpack.js]: Oooh right. I felt like I am doing this more complex that it needs to be :)

Comment (pschoffer) [reply ci/update-jetpack.js]: fixed

File: ci/update-jetpack.js (hunk 2 above)
Comment (pschoffer) [reply ci/update-jetpack.js:30]: I actually think the explicit casting there is more easily readable than the implicit one. Or is there some other reason for preferring the suggested way?

**Pull request merged**